index.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Happy Valley Dogs</title>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700|Oswald:400,700" rel="stylesheet" />
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <h1 class="page-title">Happy Valley Shelter</h1>
+    </header>
+    <div>
+        <h2>Adopt a Dog</h2>
+        <p>
+            If you are interested in adopting one of our animals, please fill out our
+            <a href="#">online adoption form</a>.
+        </p>
+    </div>
+    
+    <div class="dogs-available">
+        <div class="dog-details">
+            <img class="dog-pic" src="starter_files/images/beans.jpg">
+            <h3>Beans</h3>
+            <ul class="dog-traits-list">
+                <li style="display:inline" class="dog-trait">Adult</li>
+                <li style="display:inline" class="dog-trait">Male</li>
+                <li style="display:inline" class="dog-trait">Medium</li>
+                <li style="display:inline" class="dog-trait">Easygoing</li>
+                <li style="display:inline" class="dog-trait">Good with cats</li>
+            </ul>
+        </div>
+
+        <div class="dog-details">
+            <img class="dog-pic" src="starter_files/images/boston-terrier.jpg">
+            <h3>Zelda</h3>
+            <ul class="dog-traits-list">
+                <li style="display:inline" class="dog-trait">Adult</li>
+                <li style="display:inline" class="dog-trait">Female</li>
+                <li style="display:inline" class="dog-trait">Small</li>
+                <li style="display:inline" class="dog-trait">Easygoing</li>
+                <li style="display:inline" class="dog-trait">Good with kids</li>
+                <li style="display:inline" class="dog-trait">Good with other dogs</li>
+                <li style="display:inline" class="dog-trait"></li>
+                <li style="display:inline" class="dog-trait"></li>
+            </ul>
+        </div>
+
+        <div class="dog-details">
+            <img class="dog-pic" src="starter_files/images/boxer.jpg">
+            <h3>Baby</h3>
+            <ul class="dog-traits-list">
+                <li style="display:inline" class="dog-trait">Young</li>
```

Comment:
@nelsonblueml good work laying this out. In general, you want to avoid inline style, which is what you've done here using the style attribute. Instead, you want to apply the style to the class that you've given each <li>, `dog-trait`